code/go/0chain.net/smartcontract/zcnsc/authorizer.go

```diff
@@ -80,21 +78,22 @@ func (zcn *ZCNSmartContract) AddAuthorizer(
 		return "", err
 	}
 
+	// Info: temporarily disabling fee check because fee is still 0 on
```

Comment:
it is not a fee, it is a check before pool creation. if we comment it we will allow 0 staked pools

Reply:
This code shouldn't be here.
This check is irrelevant.
StakeLock happens here:
https://github.com/Dmdv/0chain/blob/fdb05140743c7a450b6025ce462b557dec5bd6c0/code/go/0chain.net/smartcontract/stakepool/lock.go#L44